apps/token-manager/app/src/components/Panels/AssignVotePanelContent.js

```diff
@@ -137,7 +137,7 @@ class AssignVotePanelContent extends React.Component {
       <div>
         <form onSubmit={this.handleSubmit}>
           <InfoMessage
-            title="Token Manager action"
+            title="Tokens action"
```

Comment:
This sounds a bit weird

<img width="421" alt="Screen Shot 2019-05-24 at 4 22 59 PM" src="https://user-images.githubusercontent.com/4166642/58334616-398c1800-7e40-11e9-9287-523337b94fe1.png">

cc @dizzypaty

Reply:
Yeah, just Action is fine